src/submission/validation-clinical/therapy.ts

```diff
@@ -199,30 +206,31 @@ const validateRadiationRecords = async (
     const treatmentDonorIdMatch = submittedTherapyDonorId === relatedTreatmentDonorId;
 
     const submittedTreatmentDonorIdMatch =
-      submissionTreatmentIdMatch?.submitter_donor_id === relatedTreatmentDonorId;
+      submissionTreatmentIdMatch?.submitter_donor_id === submittedTherapyDonorId;
 
     const storedTreatmentDonorIdMatch =
-      storedTreatmentIdMatch?.clinicalInfo?.submittedTherapyDonorId === relatedTreatmentDonorId;
+      storedTreatmentIdMatch?.clinicalInfo?.submittedTherapyDonorId === submittedTherapyDonorId;
 
-    const previousTreatmentDonorId =
-      relatedTreatmentDonorId ||
-      submissionTreatmentIdMatch?.submitter_donor_id ||
-      storedTreatmentIdMatch?.clinicalInfo?.submittedTherapyDonorId;
+    const previousTreatmentDonorIdMatch =
```

Comment:
@demariadaniel what does `previousTreatmentDonorId` refer to?

Reply:
We compare the currently submitted therapy record's treatment ID against 
a) other treatment on the current donor `treatmentDonorIdMatch` 
b) other treatment records that were submitted `submittedTreatmentDonorIdMatch` 
c) other treatment records in the db `storedTreatmentDonorIdMatch`